Loop/Views/ManualEntryDoseView.swift

```diff
@@ -0,0 +1,270 @@
+//
+//  ManualEntryDoseView.swift
+//  Loop
+//
+//  Created by Pete Schwamb on 12/29/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Combine
+import HealthKit
+import SwiftUI
+import LoopKit
+import LoopKitUI
+import LoopUI
+
+
+struct ManualEntryDoseView: View {
```

Comment:
[nit] Just a random thought here, but this shares a lot of similarity to `BolusEntryView` and `SimpleBolusView`.  Someday it would be nice to unify them...